packages/clerk-js/src/ui/utils/useFormControl.ts

```diff
@@ -195,19 +195,9 @@ export const useFormControlFeedback = (opts: DebouncingOption): DebouncedFeedbac
     hasPassedComplexity = false,
   } = opts;
 
-  const canDisplayFeedback = useMemo(() => {
```

Comment:
This cannot be deleted. As it affects the UI and when errors are displayed.

Here is the result, and the bug that got introduced by this change

https://github.com/clerkinc/javascript/assets/19269911/0bfbeb6d-336b-476c-953a-a192c6d21e9e

Also this is the code that creates the confirm password field. Note the `enableErrorAfterBlur` that indicates the error should only be visible after the user has left the field at least once,

```ts
  const confirmField = useFormControl('confirmPassword', '', {
    type: 'password',
    label: localizationKeys('formFieldLabel__confirmPassword'),
    isRequired: true,
    enableErrorAfterBlur: true,
  });
```

Reply:
@panteliselef nice catch. I reverted this change and took another approach.